styles/style.css

```diff
@@ -0,0 +1,572 @@
+}
+
+.product__header {
+  grid-area: 2;
+  text-align: center;
+  padding-right: .5em;
+}
+
+.product__items {
+  grid-area: 3;
+  display: flex;
+  justify-content: center;
+}
+
+.product__item {
+  display: grid;
+  height: 18.3em;
+  width: 22.24em;
+  margin-left: 1.2em;
+  position: relative;
+}
+
+.product__item:nth-child(2),
+.product__item:nth-child(3),
+.product__item:nth-child(4) { display: none; }
+
+.product__link {
+  display: grid;
+  grid-template-rows: 14.9em 1.6em;
+}
+
+.product__clrs {
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  width: 6.2em;
+  justify-self: end;
+  align-self: end;
+  align-items: center;
+}
+
+.product__img {
+  width: 13.55em;
+  height: 12.8em;
+  justify-self: center;
+}
+
+.product__clr {
+  width: 1.25em;
+  height: 1.25em;
+  border-radius: 50%;
+}
+
+.product__clr_red { background: #b62e3c; }
+
+.product__clr_bl { background: #0e1211; }
+
+.product__clr_gr { background: #c4c4c4; }
+
+.product__clr_yl { background: #fdb431; }
+
+.product__clr_cm { background: #e29d69; }
+
+.product__clr_br { background: #762e31; }
+
+.product__clr_blu { background: #41479b; }
+
+.product__clr_lb { background: #257da9; }
+
+/* .footer {height: 56.125rem;} */
+
+@media only screen and (min-width: 375px) {
+  .header__title { font-size: calc(7.63vw - .719em); }
+
+  .header__link,
+  .story__link { font-size: calc(.364em + 2.035vw); }
+
+  .product__header,
+  .info__header,
+  .stories__header { font-size: calc(.87em + 2.54vw); }
+
+  .product__name { font-size: calc(.85em + 1.27vw); }
+
+  .product__price,
+  .story__header { font-size: calc(1.27em + 1.27vw); }
+
+  .body > * { padding: 0 calc(3.23rem - 1vw); }
+
+  .nav { grid: calc(2.02rem + 10.57vw) / auto auto; }
+
+  .nav__logo_mob { width: calc(3.38rem + 13.27vw); }
+
+  .nav__bar {
+    width: calc(1.37rem + 5vw);
+    height: calc(.14rem + .8vw);
+  }
+
+  .header {
+    height: calc(57.06vw - 1.43rem);
+
+    /* grid-template-rows: 10em 16em; */
+    grid-template-rows: calc(1em + 14vw) calc(29.61vw - 2.96em);
+
+    /* background: url(../logos/rob-potter-422534-unsplash.png) no-repeat -5.2em/cover; */
+    background: url(../logos/rob-potter-422534-unsplash.png) no-repeat calc(-.14em - 16.18vw)/cover;
+  }
+
+  /* .product {height: 41.6em;} */
+  .product {
+    height: calc(18.74em + 35.73vw);
+
+    /* grid-template-rows: 5.1em min-content;
+    grid-template-rows: calc(.6em + 7.02vw) 5.5em; */
+    grid-template-rows: calc(.6em + 7.02vw) calc(10.94vw - 1.5em);
+  }
+
+  .product__items {
+    /* padding: 0 3.2em; */
+    padding: 0 calc(9.77vw - 3.05em);
+  }
+
+  .product__item {
+    /* height: 25.33em;
+    width: 21.31em; */
+    height: calc(11.6em + 21.46vw);
+    width: calc(23.13em - 2.84vw);
+    margin-left: calc(2.34em - 3.66vw);
+  }
+
+  .product__link {
+    /* grid-template-rows: 20.4em 3em; */
+    grid-template-rows: calc(9.65em + 16.79vw) calc(.26em + 4.27vw);
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)